src/explore-education-statistics-frontend/src/pages/privacy-notice.tsx

```diff
@@ -4,9 +4,11 @@ import React from 'react';
 
 function PrivacyNoticePage() {
   return (
-    <Page breadcrumbs={[{ name: 'Privacy notice' }]}>
-      <PageTitle title="Explore education statistics privacy notice" />
-
+    <Page
+      title="Explore education statistics: Privacy notice"
```

Comment:
I really don't like this title as it's unnecessarily wordy (the user knows they're on EES). Would either confirm with Fred that this is correct OR just change it and let him change it back if he's bothered about it.